functions/src/image/fetch-image/fetch-image.test.ts

```diff
@@ -0,0 +1,76 @@
+import * as admin from 'firebase-admin'
+import * as test from 'firebase-functions-test'
+import { Collections } from '../../model/collections'
+import * as functions from '../../index'
+
+describe('FetchImage', () => {
```

Comment:
There are no tests checking the order  in which the images are got. The oldest one the first one to be picked

Reply:
The `should return the next image not yet labelled by the user` and `should return the next not completed image` tests implicitly covered this scenario, but yes it makes sense to have an explicit test for this. Thanks for pointing out.

Added a new test: `should return the oldest created image as the next image to label`